Monika After Story/game/script-moods.rpy

```diff
@@ -833,3 +833,57 @@ label mas_mood_relieved:
             m 1hua "Now it'll be easier to move forward with a little more confidence, right?"
             m 1eua "I'm excited to take those next steps forward with you."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_mood_database,
+            eventlabel="mas_mood_grateful",
+            prompt="...grateful.",
+            category=[store.mas_moods.TYPE_GOOD],
+            unlocked=True
+        ),
+        code="MOO"
+    )
+
+label mas_mood_grateful:
+    $ chosen_nickname = mas_get_player_nickname()
+    m 1eub "Oh? That's nice to hear!"
+
+    m "What are you grateful for, [chosen_nickname]?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "What are you grateful for, [chosen_nickname]?{fast}"
+
+        "For you.":
+            call monika_gratefulforyou
```

Comment:
Are we planning to reuse this label? If not, we can just keep the dialogue here as it's small enough that it won't be harder to read with it here.

Otherwise, might be better for naming convention so we know the primary label this comes from
```suggestion
            call mas_mood_grateful_gratefulforyou
```